File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge1.py
def isAnagram(word1, anagram):
  word_anagram = ""
  for index in range(len(word1),0,-1):
    word_anagram += word1[index-1]
  
  if word_anagram == anagram:
    return True
  else:
    return False

print(isAnagram("zorra", "arroz"))
print(isAnagram("zorra", "zorra"))
print(isAnagram("valora", "arolav"))

File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge1.go
package main

import "fmt"

func main() {
	fmt.Println(isAnagram("zorra", "arroz"))
	fmt.Println(isAnagram("zorra", "zorra"))
}

func isAnagram(word1 string, word2 string) bool {
	var wordAnagram string
	for i := len(word1); i > 0; i-- {
		wordAnagram += string(word1[i-1])
	}
	if wordAnagram == word2 {
		return true
	}

	return false
}
